### PR TITLE
Fix small memory leak in D3DUtil.cpp

### PR DESCRIPTION
**Pull request by Karaulov**
Fix memory leak :)

### CODE REVIEW

**Comment by Karaulov** [comment]
In d3d12 too.

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by JosJuice** [comment]
Please fix the lint errors. (You can either do it automatically using `clang-format` or do it manually.) And again, you need to squash the two commits.

File: Source/Core/VideoBackends/D3D/D3DUtil.cpp [hunk 1]
```diff
@@ -188,18 +188,20 @@ int CD3DFont::Init()
...
                  CLIP_DEFAULT_PRECIS, PROOF_QUALITY, VARIABLE_PITCH, _T("Tahoma"));
   if (nullptr == hFont)
     return E_FAIL;
 
+  // Create a DC and a bitmap for the font
+  HDC hDC = CreateCompatibleDC(nullptr);
+  HBITMAP hbmBitmap = CreateDIBSection(hDC, &bmi, DIB_RGB_COLORS, (void**)&pBitmapBits, nullptr, 0);
+  SetMapMode(hDC, MM_TEXT);
+
+
```
Comment (lioncash) [inline Source/Core/VideoBackends/D3D/D3DUtil.cpp]: These two extra newlines are unnecessary.

**Comment by Karaulov** [comment]
Sorry i don't know how to merge commits. :(

**Pull request closed without merging**